CMakeLists.txt

```
@@ -61,7 +67,7 @@ function(trusted_firmware_build)
     BINARY_DIR ${TFM_BINARY_DIR}
     CMAKE_ARGS -DPROJ_CONFIG=${ZEPHYR_TFM_MODULE_DIR}/trusted-firmware-m/configs/${TFM_CFGFILE}.cmake
                -DTARGET_PLATFORM=${TFM_BOARD}
-               -DBL2=True
+               -DBL2=${TFM_BL2}
```

Only some config default to: `BL2=True`, like:
https://github.com/zephyrproject-rtos/trusted-firmware-m/blob/master/trusted-firmware-m/configs/ConfigDefaultProfileS.cmake#L37
https://github.com/zephyrproject-rtos/trusted-firmware-m/blob/master/trusted-firmware-m/configs/ConfigPsaApiTestIPC.cmake#L51

Others are not defaulting to true, so I think it would be better to let `TFM` config decide the default value, and only pass on the value if defined by the user.

```suggestion
               ${TFM_BL2_ARG}
```

```